lib/cli/src/detect.test.js

```diff
@@ -1,27 +1,288 @@
+          vuetify: '1.0.0',
+        },
+        devDependencies: {
+          'vue-loader': '1.0.0',
+        },
+      },
+    },
+  },
+  {
+    name: projectTypes.VUE,
+    files: {
+      'package.json': {
+        dependencies: {
+          vue: '1.0.0',
+        },
+      },
+    },
+  },
+  {
+    name: projectTypes.EMBER,
+    files: {
+      'package.json': {
+        devDependencies: {
+          'ember-cli': '1.0.0',
+        },
+      },
+    },
+  },
+  {
+    name: projectTypes.REACT_PROJECT,
+    files: {
+      'package.json': {
+        peerDependencies: {
+          react: '1.0.0',
+        },
+      },
+    },
+  },
+  {
+    name: projectTypes.REACT_NATIVE,
+    files: {
+      'package.json': {
+        dependencies: {
+          'react-native': '1.0.0',
+        },
+        devDependencies: {
+          'react-native-scripts': '1.0.0',
+        },
+      },
+    },
+  },
+  {
+    name: projectTypes.REACT_SCRIPTS,
+    files: {
+      'package.json': {
+        devDependencies: {
+          'react-scripts': '1.0.0',
+        },
+      },
+    },
+  },
+  {
+    name: projectTypes.WEBPACK_REACT,
+    files: {
+      'package.json': {
+        dependencies: {
+          react: '1.0.0',
+        },
+        devDependencies: {
+          webpack: '1.0.0',
+        },
+      },
+    },
+  },
+  {
+    name: projectTypes.REACT,
+    files: {
+      'package.json': {
+        dependencies: {
+          react: '1.0.0',
+        },
+      },
+    },
+  },
+  {
+    name: projectTypes.ANGULAR,
+    files: {
+      'package.json': {
+        dependencies: {
+          '@angular/core': '1.0.0',
+        },
+      },
+    },
+  },
+  {
+    name: projectTypes.WEB_COMPONENTS,
+    files: {
+      'package.json': {
+        dependencies: {
+          'lit-element': '1.0.0',
+        },
+      },
+    },
+  },
+  {
+    name: projectTypes.MITHRIL,
+    files: {
+      'package.json': {
+        dependencies: {
+          mithril: '1.0.0',
+        },
+      },
+    },
+  },
+  {
+    name: projectTypes.MARIONETTE,
+    files: {
+      'package.json': {
+        dependencies: {
+          'backbone.marionette': '1.0.0',
+        },
+      },
+    },
+  },
+  {
+    name: projectTypes.MARKO,
+    files: {
+      'package.json': {
+        dependencies: {
+          marko: '1.0.0',
+        },
+      },
+    },
+  },
+  {
+    name: projectTypes.RIOT,
+    files: {
+      'package.json': {
+        dependencies: {
+          riot: '1.0.0',
+        },
+      },
+    },
+  },
+  {
+    name: projectTypes.PREACT,
+    files: {
+      'package.json': {
+        dependencies: {
+          preact: '1.0.0',
+        },
+      },
+    },
+  },
+  {
+    name: projectTypes.RAX,
+    files: {
+      '.rax': 'file content',
+      'package.json': {
+        dependencies: {
+          rax: '1.0.0',
+        },
+      },
+    },
+  },
+];
+
+describe('Detect', () => {
+  it(`should return type ${projectTypes.HTML} if html option is passed`, () => {
```

Comment:
personal preference: avoid using variables in tests. since tests are documentation, it's common to use the literal constant in the test - more readable / less indirection

Reply:
You're right, seems much better. Thanks for your comment!